packages/RadioBoxGroup/src/RadioBox.tsx

```diff
@@ -7,7 +7,15 @@ const { css, cx } = emotion;
 
 const radioBoxInput = createDataProp('radio-box-input');
 
-export const radioBoxSizes = {
+export const Size = {
+  Default: 'default',
+  Compact: 'compact',
+  Full: 'full',
+} as const;
+
+export type SizeValues = typeof Size[keyof typeof Size];
```

Comment:
You sorta established a pattern for these shared props in the RadioGroup PR – should we move these to a separate file for each component to import from?

Reply:
yup, Harry hadn't told me about the same-name fancy magic thingy

Reply:
Oh I misread the comment. Separate file would kinda make sense here...